src/action.js

```diff
@@ -50,12 +50,12 @@ module.exports = function link(config, args) {
         registerDependencyIOS(dependencyConfig.ios, project.ios);
       }
 
-      if (project.android && dependencyConfig.assets) {
+      if (project.android && dependencyConfig.assets.length > 0) {
```

Comment:
It's extra code for sure, but readability is better. Well, I'll keep it up to you, `.length > 0` is also fine for me.